<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeScrn</class>
 <widget class="QWidget" name="WelcomeScrn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Face Transformation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">LightGray</string>
  </property>
  <widget class="QLabel" name="Welcome_msg">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>551</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Welcome to Face Transformation. Please select a option for transforming your face!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>-2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Gender_button">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>450</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Age_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLabel" name="agemsg">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>610</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Young -&amp;gt; Old&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Old -&amp;gt; Young&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gendermsg">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>610</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Masc -&amp;gt; Fem&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Fem -&amp;gt; Masc&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
